src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c

```diff
@@ -405,6 +405,10 @@ static u32 check_firewall(struct platform_device *pdev, int *level)
 	if (!FW_PRIVILEGED(fw))
 		return 0;
 
+	/* TODO for error last functionality 
```

Comment:
The format of comment in Linux kernel driver should look like
/*
 * TODO for error last...
 * str...
  * ...
  */

Reply:
done